### ISSUE TITLE
Not able to pass standalonePassword as kubernetes environment variable. We have mounted secret as volume. Not able to find more information about neodash kubernetes deployments 

### ISSUE THREAD

**Issue by madeshnandu90** [neo4j-labs/neodash#424]
## Guidelines
Please note that GitHub issues are only meant for bug reports/feature requests. 

Before creating a new issue, please check whether someone else has raised the same issue. You may be able to add context to that issue instead of duplicating the report. However, each issue should also only be focussed on a _single_ problem, so do not describe new problems within an existing thread - these are very hard to track and manage, and your problem may be ignored. Finally, do not append comments to closed issues; if the same problem re-occurs, open a new issue, and include a link to the old one.

To help us understand your issue, please specify important details, primarily:

- NeoDash version: X.Y.Z
- Neo4j Database version: X.Y.Z (Community/Enterprise/Aura).

- **Steps to reproduce**
- Expected behavior
- Actual behavior

Additionally, include (as appropriate) screenshots, drawings, etc.

**Comment by brahmprakashMishra** [comment]
Yeah - I faced this issue too. Kindly update your story to include more details though. 

The problem occurs from the docker script entrypoint.sh in scripts/ folder. It overwrites any and all variables that should be passed to nginx config from kubernetes. The other compounding challenge is that kubernetes cannot modify things directly inside the nginx owned html folder. 

I am yet to figure out a clean fix for this, maybe when time permits in future. For now, I'd recommend using the shared URL. If you do need this working earlier, you should modify the entrypoint logic such that the overwrite does not happen (so play with dockerfile).
